src/imgui/ImGuiSettings.cc

```diff
@@ -363,6 +426,23 @@ void ImGuiSettings::showMenu(MSXMotherBoard* motherBoard)
 					setStyle();
 				}
 			});
+			im::Menu("Select shortcut", [&]{
+				im::Table("table", 2, ImGuiTableFlags_SizingFixedFit, [&]{
+					auto size = ImGui::CalcTextSize("some long stuff here to test formatting"sv).x;
+					ImGui::TableSetupColumn("", ImGuiTableColumnFlags_WidthFixed, 150);
+					ImGui::TableSetupColumn("", ImGuiTableColumnFlags_WidthFixed, size);
+					if (ImGui::TableNextColumn()) {
+						openShortcutPopup(ShortcutIndex::GOTO_MEMORY_ADDRESS);
+					}
+					if (ImGui::TableNextColumn()) ImGui::TextUnformatted("goto address"sv);
+					ImGui::TableNextRow();
+					if (ImGui::TableNextColumn()) {
+						openShortcutPopup(ShortcutIndex::GOTO_DISASM_ADDRESS);
+					}
+					if (ImGui::TableNextColumn()) ImGui::TextUnformatted("some long stuff here to test formatting"sv);
+				});
+
+			});
```

Comment:
In a later version this table should be drawn via loop, instead of a hard-coded enumeration.